crates/key-server/src/mvr.rs

```diff
@@ -83,6 +84,7 @@ pub(crate) async fn mvr_forward_resolution(
             let networks: HashMap<_, _> = get_from_mvr_registry(
                 mvr_name,
                 &SuiClientBuilder::default()
+                    .request_timeout(key_server_options.rpc_timeout)
                     .build_mainnet()
```

Comment:
It's a bit strange that Network::Testnet build a mainnet client. Is this the intention?